### ISSUE TITLE
Compatibility with version 0.7x

### ISSUE THREAD

**Issue by brunocalado** [MajorVictory/RetroUI-P5e#8]
Hi. You made a great module.

Do you have plans to update it to 0.7x?

Thank you.

**Comment by MajorVictory** [comment]
Yes an update is in the works. I'll be adding Active Effects where I can, and I'm trying to keep all the old item IDs.

**Comment by brunocalado** [comment]
Thank you. You did a great job!
